README.md

```diff
@@ -146,12 +166,27 @@ f| `aws_eks_security_group_name_master` | String | Define the security group nam
 | `aws_vpc_id` | String | **Existing** AWS VPC ID to use. Accepts `vpc-###` values. |
 | `aws_vpc_subnet_id` | String | **Existing** AWS VPC Subnet ID. If none provided, will pick one. (Ideal when there's only one). |
 | `aws_vpc_enable_nat_gateway` | String | Adds a NAT gateway for each public subnet. Defaults to `true`. |
-| `aws_vpc_single_nat_gateway` | String | Toggles only one NAT gateway for all of the public subnets. Defaults to `false`. |
+| `aws_vpc_single_nat_gateway` | String | Toggles only one NAT gateway for all of the public subnets. Defaults to `true`. |
 | `aws_vpc_external_nat_ip_ids` | String | **Existing** comma separated list of IP IDs if reusing. (ElasticIPs). |
 | `aws_vpc_additional_tags` | JSON | Add additional tags to the terraform [default tags](https://www.hashicorp.com/blog/default-tags-in-the-terraform-aws-provider), any tags put here will be added to vpc provisioned resources.|
 <hr/>
 <br/>
 
+#### **Action Outputs**
+| Name             | Description                        |
+|------------------|------------------------------------|
+| `aws_vpc_id` | The selected VPC ID used. |
+| `ecs_load_balancer_dns` | ECS ALB DNS Record. |
+| `ecs_dns_record` | ECS DNS URL. |
+| `ecs_sg_id` | ECS SG ID. |
+| `ecs_lb_sg_id` | ECS LB SG ID. |
+<hr/>
+<br/>
+
+
+## Helm charts
+We have aws-auth,ingress, grafana, prometheus and loki as helm charts which can be called by deployment repo to install in aws eks cluster. User can pass input like "grafana_enable", "loki_enable", "nginx_enable", "prometheus_enable" in deployment repo along with aws access information and these charts would be installed along with eks creation in aws.
```

Comment:
Can you please **highlight** or use `code format` for the tool names? And then also use `code format` for the input names. ie `grafana_enable`, `loki_enable` etc.